zuul-core/src/main/java/com/netflix/zuul/monitoring/ConnCounter.java

```diff
@@ -32,6 +32,7 @@
 /**
  * A counter for connection stats.  Not thread-safe.
  */
+@SuppressWarnings("ErroneousBitwiseExpression")
```

Comment:
oh - what does this imply?

Reply:
The later assert in this file is just comparing constants, which the linter doesn't like